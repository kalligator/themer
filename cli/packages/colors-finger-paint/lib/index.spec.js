const { colors } = require('./index');

describe('themer color set', () => {
  it('should provide complete dark and light themes', () => {
    expect(colors).toMatchSnapshot();
  });
});
