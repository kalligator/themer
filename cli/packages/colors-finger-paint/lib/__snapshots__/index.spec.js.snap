// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`themer color set should provide complete dark and light themes 1`] = `
Object {
  "dark": Object {
    "accent0": "#C65967",
    "accent1": "#D3873B",
    "accent2": "#E5BE31",
    "accent3": "#97B55A",
    "accent4": "#76A377",
    "accent5": "#6484B5",
    "accent6": "#995785",
    "accent7": "#AC7155",
    "shade0": "#2A2021",
    "shade7": "#EDEBEE",
  },
  "light": Object {
    "accent0": "#BA394A",
    "accent1": "#D3833F",
    "accent2": "#BA991A",
    "accent3": "#78992A",
    "accent4": "#509B6B",
    "accent5": "#485A9F",
    "accent6": "#813D72",
    "accent7": "#A1684F",
    "shade0": "#EDEBEE",
    "shade7": "#2A2021",
  },
}
`;
